nvchecker_source/go.py

```diff
@@ -33,4 +33,4 @@ async def get_version_impl(info) -> VersionResult:
         version = elements[0]
         return version
     except IndexError:
-        raise GetVersionError(pkg_name, "parse error.")
```

Comment:
No, it's not the intended way. It should be

```
GetVersionError("parse error", name=pkg_name)
```

Reply:
Thanks
It has been fixed.